.kokoro/environment_tests.sh

```diff
@@ -27,6 +27,7 @@ fi
 # make sure submodule is up to date
 cd "$PROJECT_ROOT"
 git submodule update --init --recursive
+git config --global --add safe.directory '*'
```

Comment:
It looks like this should be moved up a line. It needs to run before `git submodule update`

Reply:
oh! good catch. I'll modify this.

Reply:
It looks like this fixed that issue, but there's one more change you'll need to pull in from the `main` branch. It looks like the `python3.6` references in this file need to be changed to `python3` (3.6 is too out of date for the test container)

![image](https://user-images.githubusercontent.com/3914119/195461638-ab24a4a5-8f63-446f-bba5-348d94aadfe6.png)

Reply:
I'm not sure about the failing `GraalVM` tests though

Reply:
thanks @daniel-sanche for comment - indeed, this seems as an issue!
Asd for `GraalVM`, I believe for now those can be ignored

Reply:
Applying this change seemed to have worked!